CommandsAndAdaptors/command_factory.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sys
+sys.path.append(os.path.join(str(os.getcwd()).replace("\\CommandsAndAdaptors", ''), "Functions"))
+from create_command import *
+
+# from set_command import *
+# from get_command import *
+# from delete_command import *
+
+
+class CommandFactory:
+    def __init__(self, args):
+        self.command = args[0]
+        self.validate()
+        self.schema = args[1]
+        self.database = args[2]
```

Comment:
Don’t use args[number]. It is very very bad
Use args.schema_path
Args.primary_key